packages/editor-ui/src/components/RunData.vue

```diff
@@ -840,7 +847,31 @@ export default defineComponent({
 			return Boolean(this.subworkflowExecutionError);
 		},
 		hasRunError(): boolean {
-			return Boolean(this.node && this.workflowRunData?.[this.node.name]?.[this.runIndex]?.error);
+			const hasError = Boolean(
+				this.node && this.workflowRunData?.[this.node.name]?.[this.runIndex]?.error,
+			);
+
+			if (hasError) {
```

Comment:
Computed value hasRunError() will be executed multiple times by the component, which can potentially lead to mulitple duplicate error reports. Is there a better place for sending telemetry error?